installation/docker/dockercompose.yml

```diff
@@ -19,6 +20,10 @@ services:
       - type: volume
         source: apiserver-db-data
         target: /data
+    healthcheck:
+      test: ["CMD", "nc", "-z", "127.0.0.1", "8888"]
+      interval: 10s
+      retries: 60
```

Comment:
Do we not have the tools in the container to check the healthz endpoint? If we can query `http://127.0.0.1:8081/healthz/ready` then a 200 means the server is ready, anything else means its unhealthy

Reply:
Totally. @paralta I remember that we agreed to use `curl` for testing services which have HTTP endpoint and falling back to `nc` where we are left with checking for open service ports to determine health of the service.